test/e2e/integreatly_test.go

```diff
@@ -0,0 +1,141 @@
+
+	logf.SetLogger(logf.ZapLogger(true))
+
+	installationList := &operator.InstallationList{}
+	err := framework.AddToFrameworkScheme(apis.AddToScheme, installationList)
+	if err != nil {
+		t.Fatalf("failed to add custom resource scheme to framework: %v", err)
+	}
+	// run subtests
+	t.Run("integreatly", func(t *testing.T) {
+		t.Run("Cluster", IntegreatlyCluster)
+	})
+
+}
+
+func waitForProductDeployment(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, product, deploymentName string) error {
+	namespace := intlyNamespacePrefix + product
+	t.Logf("Checking %s:%s", namespace, deploymentName)
+
+	start := time.Now()
+	err := e2eutil.WaitForDeployment(t, f.KubeClient, namespace, deploymentName, 1, deploymentRetryInterval, deploymentTimeout)
+	if err != nil {
+		return err
+	}
+
+	end := time.Now()
+	elapsed := end.Sub(start)
+
+	t.Logf("%s:%s up, waited %d", namespace, deploymentName, elapsed)
+	return nil
+}
+
+func integreatlyWorkshopTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) error {
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		return fmt.Errorf("could not get namespace: %deploymentName", err)
+	}
+
+	// create installation custom resource
+	workshopInstallation := &operator.Installation{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "e2e-workshop-installation",
+			Namespace: namespace,
+		},
+		Spec: operator.InstallationSpec{
+			Type:             "workshop",
+			NamespacePrefix:  intlyNamespacePrefix,
+			RoutingSubdomain: "apps.example.com",
+			MasterURL:        "http://console.apps.example.com",
+			SelfSignedCerts:  true,
+		},
+	}
+	// use TestCtx's create helper to create the object and add a cleanup function for the new object
+	err = f.Client.Create(goctx.TODO(), workshopInstallation, &framework.CleanupOptions{TestContext: ctx, Timeout: installationCleanupTimeout, RetryInterval: installationCleanupRetryInterval})
+	if err != nil {
+		return err
+	}
+
+	//Auth Stage - verify operators deploy
+	err = waitForProductDeployment(t, f, ctx, "rhsso", "keycloak-operator")
+	if err != nil {
+		return err
+	}
+	//Product Stage - verify operators deploy
+	products := map[string]string{
+		"3scale":               "3scale-operator",
+		"amq-online":           "enmasse-operator",
+		"amq-streams":          "amq-streams-cluster-operator",
+		"codeready-workspaces": "codeready-operator",
+		"fuse":                 "syndesis-operator",
+		"launcher":             "launcher-operator",
+		"nexus":                "nexus-operator",
+		"solution-explorer":    "tutorial-web-app-operator",
+		"user-sso":             "keycloak-operator",
+	}
+	for product, deploymentName := range products {
+		err = waitForProductDeployment(t, f, ctx, product, deploymentName)
+		if err != nil {
+			break
+		}
+	}
+	//These test only that the operators themselves have came up. Further testing should be done to verify the individual parts of each product come up also (What the operator is creating).
+
+	return err
+}
```

Comment:
This needs to be expended to do some additional testing of each operators function. Currently it will check that all product operator deployments succeed.